app/helpers/application_helper.rb

```diff
@@ -180,6 +180,33 @@ def universal_viewer_path(resource)
     "/viewer#?manifest=#{manifest_url(resource)}"
   end
 
+  def pdf_path(resource)
+    original_pdf_path(resource) || figgy_pdf_path(resource)
+  end
+
+  def original_pdf_path(resource)
+    first_member = resource.decorate.try(:members)&.first
```

Comment:
This is a bummer, this'd be really expensive. We'll want to keep our eyes on this and maybe implement a custom query to check members for a mime type. Maybe add a `#source_pdf` to wayfinder?

Reply:
Yes, I was thinking about how to avoid that. I went ahead and added `#first_member` and `#source_pdf` to the base wayfinder because they could be used by more than half of the wayfinders.